tests/lib/helpers.py

```diff
@@ -0,0 +1,15 @@
+import pytest
+import logging
+import decorator
+
+def disable_loganalyzer(func):
+    """
+    Decorator to disable loganalyzer analysis after test case execution
+    """
+    def wrapper(func, *args, **kwargs):
+        logging.debug("Disabling loganalyzer...")
+        try:
+            func(*args, **kwargs)
+        finally:
+            pytest.disable_loganalyzer = True

```

Comment:
Does it mean that one test case which uses this decorator disables loganalyzer globally regardless of subsequent test cases?

Reply:
Yes. It sets "disable_loganalyzer" attribute to true which is globally accessible. Then this attribute is handled by loganalyzer fixture.

Reply:
Used different approach. Not relevant.